prac_08/convert_miles_km.kv

```diff
@@ -0,0 +1,31 @@
+BoxLayout:
+    orientation: 'vertical'
+
+    BoxLayout:
+        orientation: 'horizontal'
+        TextInput:
+            id: input_miles
+            font_size: 48
+
```

Comment:
to make this program dynamic, you could have an on text release that automatically calculates once you have typed in your number